core/blockchain_sethead_test.go

```diff
@@ -35,6 +35,7 @@ import (
 )
 
 // rewindTest is a test case for chain rollback upon user request.
+//nolint:unused
```

Comment:
cc @karalabe a lot of code here that we don't use -- perhaps we should make use of it?